subprojects/common/junit-utils/src/main/java/ru/avito/util/RandomUtils.kt

```diff
@@ -45,7 +47,7 @@ fun randomDouble(): Double {
 }
 
 fun randomDouble(min: Double, max: Double): Double {
-    return min + (max - min) * random.nextDouble()
+    return random.nextDouble(min, max)
```

Comment:
@return a pseudorandom {@code double} value between the `min`= origin(inclusive) and the `max`= bound (exclusive)